src/cli/cli.cc

```diff
@@ -4751,7 +4797,7 @@ int main (const int argc, const char* argv[]) {
 
       // Check for gradle in pwd. Don't fail, this is just for support.
       if (!fs::exists(String("gradlew") + (platform.win ? ".bat" : ""))) {
-        log("WARN: gradlew script not in pwd: " + fs::current_path().string());
+        log("WARNING: gradlew script not in pwd: " + fs::current_path().string());
```

Comment:
we can introduce `warn` and `error` functions (later) to prefix automatically

Reply:
👍🏼